[IMP] purchase: update alternative rfq doc (#8261)
odoo/documentation: content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

==================
Alternative |RfQs|
==================

.. _purchase/manage_deals/alternative-rfqs:

.. |PO| replace:: :abbr:`PO (Purchase Order)`
.. |POs| replace:: :abbr:`POs (Purchase Orders)`
.. |RfQ| replace:: :abbr:`RfQ (Request for Quotation)`
.. |RfQs| replace:: :abbr:`RfQs (Requests for Quotation)`

Sometimes, companies might want to request offers from multiple vendors at the same time, by
inviting those vendors to submit offers for similar goods or services all at once. This helps
companies to select the cheapest, fastest vendors, depending on their specific business needs.

In Odoo, this can be done by creating alternative requests for quotation (RfQs) for different
vendors. Once a response is received from each vendor, the product lines from each |RfQ| can be
compared, and a decision can be made for which products to purchase from which vendors.

.. note::
   Sometimes referred to as a *call for tender*, this process is primarily used by organizations in
   the public sector, who are legally bound to use it when making a purchase. However, private

>    companies can also use alternative |RfQs| to spend money efficiently, as well.

Author: Nice catch. Suggestion implemented ✅

This turns the |RfQ| into a purchase order (PO), and sends an email to the vendor listed on the |PO|
form.

.. image:: calls_for_tenders/calls-for-tenders-compose-email.png
   :align: center
   :alt: Compose and send quotation email pop-up.

.. note::
   Sending emails to each vendor can be useful when creating alternative |RfQs|, because vendors can
   confirm if their past prices still hold today, which helps companies choose the best offers for
   them.

Create alternative |RfQs|
=========================

Once a |PO| is created and sent to a vendor, alternative |RfQs| can be created for additional
vendors to compare prices, delivery times, and other factors to help make a decision for the order.

To create alternative |RfQs|, from the |PO| form, click the :guilabel:`Alternatives` tab. Then,
click :guilabel:`Create Alternative`. When clicked, a :guilabel:`Create alternative` pop-up window
appears.

.. image:: calls_for_tenders/calls-for-tenders-create-alternative.png
   :align: center
   :alt: Calls for tenders pop-up to create alternative quotation.

From this window, select a new, different vendor from the drop-down menu next to the

> :guilabel:`Vendor` field to whom to assign this alternative quotation.

Author: Suggestion implemented ✅

followed by :guilabel:`Create Alternative`.

This opens the :guilabel:`Create alternative` pop-up window. Once again, choose a different vendor
from the drop-down menu next to :guilabel:`Vendor`. This time, however, *uncheck* the
:guilabel:`Copy Products` checkbox. Once ready, click :guilabel:`Create Alternative`.

.. tip::
   If an alternative quotation should be removed from the :guilabel:`Alternatives` tab, they can be
   individually removed by clicking on the :guilabel:`Remove (X)` icon at the end of their row.

This creates a third, new |PO|. But, since the product quantities of the original |PO| were **not**
copied over, the product lines are empty, and new products must be added by clicking :guilabel:`Add
a product`, and selecting the desired products from the drop-down menu.

Once the desired number of products are added, click :guilabel:`Send by Email`.

.. image:: calls_for_tenders/calls-for-tenders-blank-quotation.png
   :align: center
   :alt: Blank alternative quotation with alternatives in breadcrumbs.

This opens a :guilabel:`Compose Email` pop-up window, wherein the message to the vendor can be

> customized. Once ready, click :guilabel:`Send`.

Author: No, alternative RfQs can be created with the checkbox either checked _or_ unchecked. But I see where the confusion could lie, with this example following the one above. I added in a few sentences of clarification to try and specify that this is just the instruction for this particular example.

Suggestion implemented ✅

Sometimes, companies might want to request offers from multiple vendors at the same time, by
inviting those vendors to submit offers for similar goods or services all at once. This helps
companies to select the cheapest, fastest vendors, depending on their specific business needs.

In Odoo, this can be done by creating alternative requests for quotation (RfQs) for different
vendors. Once a response is received from each vendor, the product lines from each |RfQ| can be
compared, and a decision can be made for which products to purchase from which vendors.

.. note::
   Sometimes referred to as a *call for tender*, this process is primarily used by organizations in
   the public sector, who are legally bound to use it when making a purchase. However, private
   companies can also use alternative |RfQs| to spend money efficiently.

Configuration
=============

To create alternative |RfQs|, the *Purchase Agreements* feature **must** be enabled in the
*Purchase* app settings. To enable the feature, navigate to :menuselection:`Purchase app -->
Configuration --> Settings`. Under the :guilabel:`Orders` section, click the checkbox for

> :guilabel:`Purchase Agreements`.

Author: Suggestion implemented ✅

Doing so enables the ability to create alternative |RfQs|, as well as *blanket orders*.

.. image:: calls_for_tenders/calls-for-tenders-enabled-setting.png
   :align: center
   :alt: Purchase Agreements enabled in the Purchase app settings.

.. tip::
   To save time creating alternative |RfQs|, custom vendors, prices, and delivery lead times can be
   set in the :guilabel:`Purchase` tab of a product form.

>    Navigate to :menuselection:`Purchase app --> Products --> Products`, and select a product to

Author: Suggestion implemented ✅

   edit. On the product form, click the :guilabel:`Purchase` tab, and click :guilabel:`Add a line`.

>    From the drop-down menu, choose a vendor to set under the :guilabel:`Vendor` column, and set a

Author: Suggestion implemented ✅

   :guilabel:`Price` and :guilabel:`Delivery Lead Time`, if necessary. Clicking the

>    :guilabel:`⋮ (three dots)` icon provides additional visibility options to add to the line item.

Author: I'm gonna go with :guilabel:`(optional columns drop-down toggle) icon` - because that's what's showing in the Inspect code... I know it's wordy 😕 If I need to change it again I will...

![optional columns icon](https://github.com/odoo/documentation/assets/120495567/db43b776-4524-4fe8-9bcc-d689822ca704)

Create an |RfQ|
===============

To create a new |RfQ|, navigate to the :menuselection:`Purchase` app, and click :guilabel:`New`.

> On the blank |RfQ| form, add a vendor from the drop-down next to the :guilabel:`Vendor` field, and

Author: Suggestion implemented ✅

> click :guilabel:`Add a product` to select a product from the drop-down menu in the

Author: Suggestion implemented ✅

:guilabel:`Product` column.

Then, set the desired quantity to purchase in the :guilabel:`Quantity` column, and change the
purchase price in the :guilabel:`Unit Price` column, if necessary.

> Clicking the :guilabel:`⋮ (three dots)` icon provides additional visibility options to add to the

Author: See previous, unresolved comment addressing this. Suggestion implemented ✅

line item.

> Repeat these steps to add as many options as desired, including the :guilabel:`UoM` (Units of

Author: Suggestion implemented ✅

Measure) to purchase the products in, and the :guilabel:`Expected Arrival` date.

Once ready, click :guilabel:`Send by Email`.

This opens a :guilabel:`Compose Email` pop-up window, wherein the message to the vendor can be

> customized. Once ready, click :guilabel:`Send`.

Author: Suggestion implemented ✅

This turns the |RfQ| into a purchase order (PO), and sends an email to the vendor listed on the |PO|

> form.

Author: You are correct. Suggestion implemented ✅

.. image:: calls_for_tenders/calls-for-tenders-compose-email.png
   :align: center
   :alt: Compose and send quotation email pop-up.

.. note::
   Sending emails to each vendor can be useful when creating alternative |RfQs|, because vendors can
   confirm if their past prices still hold today, which helps companies choose the best offers for

>    them.

Author: Suggestion implemented ✅

Create alternative |RfQs|
=========================

Once a |PO| is created and sent to a vendor, alternative |RfQs| can be created for additional
vendors to compare prices, delivery times, and other factors, to help make a decision for the order.

> To create alternative |RfQs|, from the |PO| form, click the :guilabel:`Alternatives` tab. Then,

Author: You are correct - changed sentence to reflect that. Suggestion implemented ✅

click :guilabel:`Create Alternative`. When clicked, a :guilabel:`Create alternative` pop-up window
appears.

.. image:: calls_for_tenders/calls-for-tenders-create-alternative.png
   :align: center
   :alt: Calls for tenders pop-up to create alternative quotation.

From this window, select an alternative vendor from the drop-down menu next to the
:guilabel:`Vendor` field, to whom the alternative quotation is assigned.

Next to this, there is a :guilabel:`Copy Products` checkbox that is selected by default. When

> selected, the product quantities of the original |PO| are copied over to the alternative. For this

Author: You are correct - changed sentence to reflect that. Suggestion implemented ✅

first alternative quotation, leave the checkbox checked. Once finished, click :guilabel:`Create

> Alternative`. This opens a new |PO|.

Author: You are correct - changed sentence to reflect that. Suggestion implemented ✅

Since the :guilabel:`Create Alternative` checkbox was left checked, the new form is already

> pre-populated with the same products, quantities, and other details as the previous, original |PO|.

Author: You are correct - changed sentence to reflect that. Suggestion implemented ✅

.. note::
   When the :guilabel:`Copy Products` checkbox is selected while creating an alternative quotation,
   additional products do **not** need to be added unless desired.

   However, if a chosen vendor is listed in the :guilabel:`Vendor` column under a specific product

>    form included in the order, the values set on the product form carry over to the |PO|, and

Author: Suggestion implemented ✅

   **must** be changed manually, if necessary.

Once ready, create a second alternative quotation by clicking the :guilabel:`Alternatives` tab,

> followed by :guilabel:`Create Alternative`.

Author: Don't want to explain it in full here, because there's a dedicated section to it later on. As a compromise I added this note in:

.. note::
   An option to :guilabel:`Compare Product Lines` also appears. Clicking this navigates to the
   :guilabel:`Compare Order Lines` page, where alternative |RfQs| can be compared, side-by-side. For
   more information on this, refer to the :ref:`Compare Product Lines
   <purchase/manage_deals/compare-product-lines>` section.

Suggestion implemented ✅

This opens the :guilabel:`Create alternative` pop-up window. Once again, choose a different vendor
from the drop-down menu next to :guilabel:`Vendor`. For this particular |RfQ|, however, *uncheck*

> the :guilabel:`Copy Products` checkbox. Doing so removes all products on the new alternative |RfQ|,

Author: Expanded upon this sentence, and I feel like it explains why:

For this particular |RfQ|, however, *uncheck*
the :guilabel:`Copy Products` checkbox. Doing so removes all products on the new alternative |RfQ|,
leaving it blank. The specific products which should be ordered from this particular vendor can be
added in as needed.

Suggestion implemented ✅

leaving it blank. Products can be added in as needed.

Once ready, click :guilabel:`Create Alternative`.

.. tip::
   If an alternative quotation should be removed from the :guilabel:`Alternatives` tab, they can be
   individually removed by clicking on the :guilabel:`Remove (X)` icon at the end of their row.

This creates a third, new |PO|. But, since the product quantities of the original |PO| were **not**
copied over, the product lines are empty, and new products must be added by clicking :guilabel:`Add

> a product`, and selecting the desired products from the drop-down menu.

Author: Expanded upon this above, and I feel like it explains why:

For this particular |RfQ|, however, uncheck
the :guilabel:Copy Products checkbox. Doing so removes all products on the new alternative |RfQ|,
leaving it blank. The specific products which should be ordered from this particular vendor can be
added in as needed.

Suggestion implemented ✅

Once the desired number of products are added, click :guilabel:`Send by Email`.

.. image:: calls_for_tenders/calls-for-tenders-blank-quotation.png
   :align: center
   :alt: Blank alternative quotation with alternatives in breadcrumbs.

This opens a :guilabel:`Compose Email` pop-up window, wherein the message to the vendor can be

> customized. Once ready, click :guilabel:`Send`.

Author: Suggestion implemented ✅

From this newest form, click the :guilabel:`Alternatives` tab. Under this tab, all three |POs| can
be seen in the :guilabel:`Reference` column. Additionally, the vendors are listed under the
:guilabel:`Vendor` column, and the order :guilabel:`Total` and :guilabel:`Status` of the orders are
in the rows, as well.

> Link a new |RfQ| to existing quotations

Author: Suggestion implemented ✅

=======================================

Even if a quotation is not created directly from the :guilabel:`Alternatives` tab of another |RfQ|,
it can still be linked to existing |RfQs|.

> Begin with creating a new |RfQ|, by navigating to :menuselection:`Purchase app --> New`. Fill out

Author: Suggestion implemented ✅

> the |RfQ| according to the earlier instructions.

Author: Added the ref tag 👍 Suggestion implemented ✅

> Then, once ready, click the :guilabel:`Alternatives` tab. Since this new |PO| was created

Reviewer: RfQ not a PO, right?

Just trying to make sure things are as clear as possible for the reader to follow along with.  So, if an RfQ is 'technically' a PO, just in its early stages, it still might be beneficial to keep that same, super-accurate terminology in tact, so as to avoid any potential confusion.

Author: You are correct. Made the change to reflect that -Suggestion implemented ✅

separately, there are no other orders linked yet.

> To link this order with alternatives, click :guilabel:`Link to Existing RfQ` on the first line in

Reviewer: ```suggestion
However, to link this RfQ with existing alternatives, click :guilabel:`Link to Existing RfQ` on the first line in
```

Author: Suggestion implemented ✅

the :guilabel:`Vendor` column.

.. image:: calls_for_tenders/calls-for-tenders-link-rfq-popup.png
   :align: center
   :alt: Pop-up to link new quotation to existing RFQs.

> This opens an :guilabel:`Add: Alternative POs` pop-up window. Select the three previously-created

Reviewer: ```suggestion
This opens an :guilabel:`Add: Alternative POs` pop-up window. Select the desired previously-created
```

Author: Suggestion implemented ✅

> |POs|, and click :guilabel:`Select`. All of these orders are now copied to this |PO|, and can be

Reviewer: RfQs, not POs again...right?

Author: Right. Suggestion implemented ✅

found under the :guilabel:`Alternatives` tab.

.. tip::

>    If a large number of |POs| are being processed and the previous |POs| can't be located, click

Reviewer: ```suggestion
   If a large number of |POs| are being processed, and the previous |POs| can't be located, click
```

Author: Suggestion implemented ✅

>    :menuselection:`Group By --> Vendor` (under the search bar, at the top of the pop-up window) to

Reviewer: I think you'll need to be specific about how they'll need to open the search bar drop-down menu of filter options (including an emoji of the arrow being clicked, if an accurate one exists), and then get into the 'Group By --> Vendor' instructions.

Author: Expanded this paragraph to be very clear about how to do this. Suggestion implemented ✅

>    group the vendors selected on the previous orders.

Reviewer: ```suggestion
   group the results by the vendors selected on the previous orders.
```

Author: Suggestion implemented ✅

Compare product lines
=====================

> Alternative |RfQs| can be compared side-by-side in order to determine which vendors offer the best

Reviewer: ```suggestion
Alternative |RfQs| can be compared side-by-side, in order to determine which vendors offer the best
```
I agree that RfQs should be used like this (as it seems the most appropriate), but the lack of common usage throughout the doc may make this doc hard to follow for some readers, so try to make sure the most accurate terms are used uniformly throughout, please. Thanks! 👍

deals on the products included in the orders.

To compare alternative |RfQs|, navigate to the :menuselection:`Purchase` app, and select one of the
previously-created |RfQs|.

> Click the :guilabel:`Alternatives` tab to see all linked |RfQs|. Then, under the :guilabel:`Create

Reviewer: ```suggestion
Then, click the :guilabel:`Alternatives` tab to see all linked |RfQs|. Next, under the :guilabel:`Create
```

Author: Suggestion implemented ✅

> Alternative` tab, click :guilabel:`Compare Product Lines`. This navigates to the :guilabel:`Compare

Reviewer: ```suggestion
Alternative` option, click :guilabel:`Compare Product Lines`. This navigates to the :guilabel:`Compare
```

Order Lines` page.

.. image:: calls_for_tenders/calls-for-tenders-compare-products.png
   :align: center
   :alt: Compare Product Lines page for alternative RFQs.

The :guilabel:`Compare Order Lines` page, by default, groups by :guilabel:`Product`. Each product

> included in any of the |RfQs| is displayed in its own drop-down, and lists all of the |PO| numbers

Reviewer: ```suggestion
included in any of the |RfQs| is displayed in its own nested drop-down list, and features all of the |PO| numbers
```

Author: Suggestion implemented ✅

in the :guilabel:`Reference` column.

Additional columns on this page include the :guilabel:`Vendor` from which products were ordered,

> the :guilabel:`Status` of the quotation (e.g., RFQ, RFQ Sent), the :guilabel:`Quantity` of products

Reviewer: ```suggestion
the :guilabel:`Status` of the quotation (e.g., :guilabel:`RFQ`, :guilabel:`RFQ Sent`, etc.), the :guilabel:`Quantity` of products
```

Author: Suggestion implemented ✅

ordered from each vendor, the :guilabel:`Unit Price` per product, and the :guilabel:`Total` price of

> the order.

Reviewer: I'm also seeing columns for: OTD, Description, Expected Arrival, Unit of Measure, and Currency. Are these *not* by default? Either way, it would be wise to double-check to see which columns *do* appear by default, just to make sure everything about this doc is accurate as possible. 👍

Author: Added in each of these columns. Suggestion implemented ✅

.. note::
   To remove product lines from the :guilabel:`Compare Order Lines` page, click :guilabel:`Clear` at
   the far-right end of that product line's row.

   Doing so removes this specific product as a selectable option from the page, and changes the

>    :guilabel:`Total` price of that product on the page to **0**.

Reviewer: ```suggestion
   :guilabel:`Total` price of that product on the page to `0`.
```

Author: Suggestion implemented ✅

>    Additionally, on the |PO| form in which that product was included, its ordered quantity is also

Reviewer: ```suggestion
   Additionally, on the |RfQ| form, in which that product was included, its ordered quantity is also
```

Author: Suggestion implemented ✅

>    changed to **0**.

Reviewer: ```suggestion
   changed to `0`.
```

Author: Suggestion implemented ✅

Once the best offers have been identified, individual products can be selected by clicking the
:guilabel:`Choose` button at the end of each corresponding row.

Once all desired products have been chosen, click :guilabel:`Requests for Quotation` (in the
breadcrumbs, at the top of the page) to navigate back to an overview of all |RfQs|.

Cancel (or keep) alternatives
=============================

Once the desired products have been chosen from the :guilabel:`Compare Order Lines` page, the

> remaining |RfQs| from which no products were chosen can be canceled.

Reviewer: ```suggestion
remaining |RfQs|, from which no products were chosen, can be canceled.
```

Author: Suggestion implemented ✅

The cost in the :guilabel:`Total` column for each product that wasn't chosen is automatically set to

> **0**, indicated at the far-right of each corresponding row.

Reviewer: ```suggestion
`0`, indicated at the far-right of each corresponding row.
```

Author: Suggestion implemented ✅

Although they haven't been canceled yet, this indicates that each of those orders can be canceled
without having an effect on the other live orders, once those orders have been confirmed.

.. image:: calls_for_tenders/calls-for-tenders-zero-total.png
   :align: center
   :alt: Canceled quotations in the Purchase app overview.

> To confirm an |RfQ| for which products **were** selected, click into one, and click

Reviewer: ```suggestion
To confirm an |RfQ| for which products were selected, click into one, and click
```

Author: Suggestion implemented ✅

:guilabel:`Confirm Order`.

This causes a pop-up window reading :guilabel:`What about the alternative Requests for Quotations?`
to appear. From the pop-up window, two options are presented: :guilabel:`Cancel Alternatives` and
:guilabel:`Keep Alternatives`.

If this |PO| should **not** be confirmed, click :guilabel:`Discard`.

Selecting :guilabel:`Cancel Alternatives` automatically cancels the alternative |RfQs|. Selecting

> :guilabel:`Keep Alternatives` keeps the alternative |RfQs| open, so they can still be accessed if

Reviewer: ```suggestion
:guilabel:`Keep Alternatives` keeps the alternative |RfQs| open, so they can still be accessed, if
```

Author: Suggestion implemented ✅

Once all products are ordered, :guilabel:`Cancel Alternatives` can be selected from whichever |PO|
is open at that time.

.. image:: calls_for_tenders/calls-for-tenders-keep-or-cancel.png
   :align: center
   :alt: Keep or cancel pop-up for alternative RFQs.

To view a detailed form of one of the |RfQs| listed, click the line item for that quotation. This
opens an :guilabel:`Open: Alternative POs` pop-up window, from which all details of that particular
|RfQ| can be viewed.

Once ready, click :guilabel:`Close` to close the pop-up window.

If all alternative |RfQs| should stay open, from the :guilabel:`What about the alternative Requests
for Quotations?` pop-up window, click :guilabel:`Keep Alternatives`.

Then, click :guilabel:`Requests for Quotation` (in the breadcrumbs, at the top of the page) to
navigate back to an overview of all |RfQs|.

Click into the remaining |RfQs| that contain products that need to be ordered, and click
:guilabel:`Confirm Order`.

This opens the :guilabel:`What about the alternative Requests for Quotations?` pop-up window. This
time, click :guilabel:`Cancel Alternatives` to cancel all other alternative |RfQs| linked with this

> order.

Reviewer: What's the purpose behind this? A little clarification on why they would need to do this could be helpful.

Author: Clarified the reasoning behind it in this sentence. Suggestion implemented ✅

> ==================
> Alternative |RfQs|
> ==================

Reviewer 2: While the |shorthand| does work in the H1, I'd like to leave it out just in case it doesn't format properly :)
```suggestion
================
Alternative RfQs
================
```

Author: Suggestion implemented ✅

Configuration
=============

To create alternative |RfQs|, the *Purchase Agreements* feature **must** be enabled in the
*Purchase* app settings. To enable the feature, navigate to :menuselection:`Purchase app -->
Configuration --> Settings`. Under the :guilabel:`Orders` section, click the checkbox for
:guilabel:`Purchase Agreements`.

Then, click :guilabel:`Save` to apply the change.

Doing so enables the ability to create alternative |RfQs|, as well as *blanket orders*.

.. image:: calls_for_tenders/calls-for-tenders-enabled-setting.png
   :align: center
   :alt: Purchase Agreements enabled in the Purchase app settings.

.. tip::
   To save time creating alternative |RfQs|, custom vendors, prices, and delivery lead times can be
   set in the :guilabel:`Purchase` tab of a product form.

   To do that, navigate to :menuselection:`Purchase app --> Products --> Products`, and select a
   product to edit. On the product form, click the :guilabel:`Purchase` tab, and click
   :guilabel:`Add a line`.

   From the drop-down menu, choose a vendor to set under the :guilabel:`Vendor` column, then set a
   :guilabel:`Price` and :guilabel:`Delivery Lead Time`, if necessary. Clicking the

>    :guilabel:`(optional columns drop-down toggle)` icon provides additional column options to add to

Reviewer: perhaps it would helpful to always mention where the  user can find this icon...

Author: Added clarification for where to find this icon. Suggestion implemented ✅

   the line item.

.. _purchase/manage_deals/create-rfq:

Create an |RfQ|
===============

To create a new |RfQ|, navigate to the :menuselection:`Purchase` app, and click :guilabel:`New`.

On the blank |RfQ| form, add a vendor from the drop-down menu, next to the :guilabel:`Vendor` field.
Then, click :guilabel:`Add a product`, in the :guilabel:`Products` tab, to select a product from the
drop-down menu in the :guilabel:`Product` column.

Then, set the desired quantity to purchase in the :guilabel:`Quantity` column, and change the
purchase price in the :guilabel:`Unit Price` column, if necessary.

> Clicking the :guilabel:`(optional columns drop-down toggle)` icon provides additional column options

Reviewer: same note as above

Author: Added clarification for where to find this icon. Suggestion implemented ✅

To create alternative |RfQs| from the original, click the :guilabel:`Alternatives` tab. Then, click
:guilabel:`Create Alternative`. When clicked, a :guilabel:`Create alternative` pop-up window
appears.

.. image:: calls_for_tenders/calls-for-tenders-create-alternative.png
   :align: center
   :alt: Calls for tenders pop-up to create alternative quotation.

From this window, select an alternative vendor from the drop-down menu next to the
:guilabel:`Vendor` field, to whom the alternative quotation is assigned.

Next to this, there is a :guilabel:`Copy Products` checkbox that is selected by default. When
selected, the product quantities of the original |RfQ| are copied over to the alternative. For this
first alternative quotation, leave the checkbox checked. Once finished, click :guilabel:`Create
Alternative`. This opens a new |RfQ| form.

Since the :guilabel:`Create Alternative` checkbox was left checked, the new form is already
pre-populated with the same products, quantities, and other details as the previous, original |RfQ|.

.. note::
   When the :guilabel:`Copy Products` checkbox is selected while creating an alternative quotation,

>    additional products do **not** need to be added unless desired.

Reviewer: ```suggestion
   additional products do **not** need to be added, unless desired.
```

Author: Suggestion implemented ✅

   form included in the order, the values set on the product form carry over to the |RfQ|, and
   **must** be changed manually, if necessary.

Once ready, create a second alternative quotation by clicking the :guilabel:`Alternatives` tab,
followed by :guilabel:`Create Alternative`.

.. note::
   An option to :guilabel:`Compare Product Lines` also appears. Clicking this navigates to the
   :guilabel:`Compare Order Lines` page, where alternative |RfQs| can be compared, side-by-side. For
   more information on this, refer to the :ref:`Compare Product Lines
   <purchase/manage_deals/compare-product-lines>` section.

This opens the :guilabel:`Create alternative` pop-up window. Once again, choose a different vendor
from the drop-down menu next to :guilabel:`Vendor`. For this particular |RfQ|, however, *uncheck*
the :guilabel:`Copy Products` checkbox. Doing so removes all products on the new alternative |RfQ|,
leaving it blank. The specific products which should be ordered from this particular vendor can be
added in as needed.

Once ready, click :guilabel:`Create Alternative`.

.. tip::
   If an alternative quotation should be removed from the :guilabel:`Alternatives` tab, they can be

>    individually removed by clicking on the :guilabel:`Remove (X)` icon at the end of their row.

Reviewer 2: Icon formatting
```suggestion
   individually removed by clicking on the :guilabel:`X (remove)` icon at the end of their row.
```

Author: Suggestion implemented ✅

This creates a third, new |RfQ|. But, since the product quantities of the original |RfQ| were
**not** copied over, the product lines are empty, and new products can be added as needed by
clicking :guilabel:`Add a product`, and selecting the desired products from the drop-down menu.

Once the desired number of specific products are added, click :guilabel:`Send by Email`.

.. image:: calls_for_tenders/calls-for-tenders-blank-quotation.png
   :align: center
   :alt: Blank alternative quotation with alternatives in breadcrumbs.

This opens a :guilabel:`Compose Email` pop-up window, wherein the message to the vendor can be
customized, and attachments can be added, if necessary. Once ready, click :guilabel:`Send`.

From this newest form, click the :guilabel:`Alternatives` tab. Under this tab, all three |RfQs| can
be seen in the :guilabel:`Reference` column. Additionally, the vendors are listed under the

> :guilabel:`Vendor` column, and the order :guilabel:`Total` and :guilabel:`Status` of the orders are

Reviewer: ```suggestion
:guilabel:`Vendor` column, and the order :guilabel:`Total` (and :guilabel:`Status`) of the orders are
```

=====================================

Even if a quotation is not created directly from the :guilabel:`Alternatives` tab of another |RfQ|,
it can still be linked to existing |RfQs|.

To do that, begin by creating a new |RfQ|. Navigate to :menuselection:`Purchase app --> New`. Fill
out the |RfQ|, according to the :ref:`previous instructions <purchase/manage_deals/create-rfq>`.

Then, once ready, click the :guilabel:`Alternatives` tab. Since this new |RfQ| was created
separately, there are no other orders linked yet.

However, to link this |RfQ| with existing alternatives, click :guilabel:`Link to Existing RfQ` on
the first line in the :guilabel:`Vendor` column.

.. image:: calls_for_tenders/calls-for-tenders-link-rfq-popup.png
   :align: center
   :alt: Pop-up to link new quotation to existing RFQs.

This opens an :guilabel:`Add: Alternative POs` pop-up window. Select the desired previously-created
|RfQs|, and click :guilabel:`Select`. All of these orders are now copied to this |RfQ|, and can be
found under the :guilabel:`Alternatives` tab.

.. tip::
   If a large number of |POs| are being processed, and the previous |POs| can't be located, click

>    the :guilabel:`⬇️ (down arrow)` icon to the right of the search bar,  at the top of the pop-up

Reviewer: ```suggestion
   the :guilabel:`⬇️ (down arrow)` icon to the right of the search bar, at the top of the pop-up
```

Author: Suggestion implemented ✅

>    the :guilabel:`⬇️ (down arrow)` icon to the right of the search bar, at the top of the pop-up

Reviewer 2: Remove non breaking space character and replace with a normal space :)
```suggestion
   the :guilabel:`⬇️ (down arrow)` icon to the right of the search bar, at the top of the pop-up
```

Author: Suggestion implemented ✅
